docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -1 +1,195 @@
-include::ROOT:page$under-construction.adoc[]
+= Plugin Web App
+
+== General Info
+
+The plugin provides functionality to ???.
+
+== Installation
+
+.build.gradle
+[source,gradle]
+----
+implementation(group: 'org.vividus', name: 'vividus-plugin-web-app', version: '0.2.0')
+----
+
+
+== Properties
+
+=== Required
+
+There are no required properties for the plugin.
+
+=== Optional
+
+There are no required optional for the plugin.
+
+
+== Steps
+
+=== Clear proxy log
+
+==== *_Info_*
+
+Clears the proxy log
+
+==== *_Wording_*
+
+[source,gherkin]
+----
+When I clear proxy log
+----
+
+==== *_Usage_*
+
+.ProxyStepsTests.story
+[source,gherkin]
+----
+Scenario: Verify step When I clear proxy log
+Given I am on a page with the URL 'http:/httpbin.org/get'
+When I clear proxy log
+Then number of HTTP GET requests with URL pattern `http://httpbin.org/get` is EQUAL TO `0`
+----
+
+
+=== Checks if the number of requests with given URL-pattern exist
+
+==== *_Info_*
+
+This step requires proxy to be turned on. It can be done via setting properties or switching on @proxy metatag.
+Step gets proxies log, extract from contained requests URLs and match them with URL-pattern
+If URLs are the same, there were calls with given URL-pattern, otherwise - weren't.
+If there weren't any calls matching requirements, HAR file with all calls will be attached to report.
+If response contains status code 302 - corresponding request will be filtered out.
+
+==== *_Wording_*
+
+[source,gherkin]
+----
+Then number of HTTP $httpMethod requests with URL pattern `$urlPattern` is $comparisonRule `$number`
+----
+
+==== *_Parameters_*
+
+. `$httpMethod` - HTTP method to filter by
+. `$urlPattern` - the string value of URL-pattern to filter by
+. `$comparisonRule` - xref:parameters:comparison-rule.adoc[Comparison Rule]
+. `$number` - The number to compare with
+
+==== *_Usage_*
+
+.ProxyStepsTests.story
+[source,gherkin]
+----
+Scenario: Verify step When I clear proxy log
+Given I am on a page with the URL 'http:/httpbin.org/get'
+When I clear proxy log
+Then number of HTTP GET requests with URL pattern `http://httpbin.org/get` is EQUAL TO `0`
+----
+
+
+=== Saves the query string from request with given URL-pattern into the variable with specified name and scopes.
+
+==== *_Info_*
+
+This step requires proxy to be turned on.
```

Comment:
you could reuse the same block of the text using attributes https://docs.antora.org/antora/2.3/page/attributes/#attribute-entry